src/olympia/reviewers/models.py

```diff
@@ -836,7 +836,7 @@ def set_on_addons_latest_signed_versions(cls, addons, reason):
         used_generator_last_iteration = None
         due_date = None
         for addon in addons:
-            if used_generator_last_iteration is not False:
+            if used_generator_last_iteration != []:
```

Comment:
I understand that `set_needs_human_review_on_latest_versions` is supposed to return a list, but why was it checking if not false in the first place? Is this just some bug or why does it need to change now to checking for an empty array?